packages/entity/src/metrics/EntityMetricsUtils.ts

```diff
@@ -44,14 +48,16 @@ export const timeAndLogLoadMapEventAsync = (
 
 export const timeAndLogMutationEventAsync = (
   metricsAdapter: IEntityMetricsAdapter,
-  mutationType: EntityMetricsMutationType
+  mutationType: EntityMetricsMutationType,
+  entityClassName: string
```

Comment:
Just wanted to ask whether ordering this by `name, type` would make more sense than `type, name` depending on what you think is more important to put first

Reply:
I thought briefly about this as I was adding it and chose `type, name` for the somewhat arbitrary reason that I liked the stricter type to precede the less strict type. That being said, I don't really have a preference unless we were to add another string argument, in which case I like to split them up with concrete types to make incorrect ordering less likely.

Reply:
Based on the fact that we added type before class, my gut says type is more important.